nextflow_schema.json

```diff
@@ -694,6 +694,214 @@
+                    "fa_icon": "fas fa-align-left",
+                    "help_text": "Use this to run the program in 'BlastN', 'BlastP', 'BlastX' modes to align DNA\nand DNA, protein and protein, or DNA reads against protein references\nrespectively. Ensure your database matches the mode. Check the\n[MALT\nmanual](http://ab.inf.uni-tuebingen.de/data/software/malt/download/manual.pdf)\nfor more details. Default: `'BlastN'`\n\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies MALT parameter: `-m`\n",
+                    "enum": ["BlastN", "BlastP", "BlastX"]
+                },
+                "metagenomics_malt_alignment_mode": {
+                    "type": "string",
+                    "default": "SemiGlobal",
+                    "description": "Specify alignment method for MALT. Options: 'Local', 'SemiGlobal'.",
+                    "fa_icon": "fas fa-align-center",
+                    "help_text": "Specify what alignment algorithm to use. Options are 'Local' or 'SemiGlobal'. Local is a BLAST like alignment, but is much slower. Semi-global alignment aligns reads end-to-end. Default: `'SemiGlobal'`\n\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies MALT parameter: `-at`",
+                    "enum": ["Local", "SemiGlobal"]
+                },
+                "metagenomics_malt_min_percent_identity": {
+                    "type": "integer",
+                    "default": 85,
+                    "description": "Percent identity value threshold for MALT.",
+                    "fa_icon": "fas fa-id-card",
+                    "help_text": "Specify the minimum percent identity (or similarity) a sequence must have to the reference for it to be retained. Default is `85`\n\nOnly used when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies MALT parameter: `-id`"
+                },
+                "metagenomics_malt_top_percent": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "Specify the percent for LCA algorithm for MALT (see MEGAN6 CE manual).",
+                    "fa_icon": "fas fa-percent",
+                    "help_text": "Specify the top percent value of the LCA algorithm. From the [MALT manual](http://ab.inf.uni-tuebingen.de/data/software/malt/download/manual.pdf): \"For each\nread, only those matches are used for taxonomic placement whose bit disjointScore is within\n10% of the best disjointScore for that read.\". Default: `1`.\n\nOnly when `--metagenomics_profiling_tool malt` is also supplied.\n\n> Modifies MALT parameter: `-top`"
+                },
+                "metagenomics_malt_min_support_mode": {
+                    "type": "string",
+                    "default": "percent",
+                    "description": "Specify whether to use percent or raw number of reads for minimum support required for taxon to be retained for MALT. Options: 'percent', 'reads'.",
```

Comment:
Just saw you have `options` in the description, you don't need this as you have the enum. Such strings can be removed from all entries as to reduce the length of `--help` messages